tests/codeception/tests/acceptance.suite.yml

```diff
@@ -22,12 +22,11 @@ modules:
                 'browserstack.user': 'rtcamp3'
                 'browserstack.key': '%BROWSERSTACK_ACCESS_KEY%'
                 'os': 'OS X'
-                'os_version': 'mojave'
-                'browser_version': '78'
+                'os_version': 'big sur'
                 'browserstack.debug': 'true'
                 'browserstack.networkLogs': 'true'
                 'build': '%TRAVIS_BUILD_NUMBER%'
-                'project': 'rtMedia Automation - 2019'
+                'project': 'rtMedia Automation - 2022'
```

Comment:
use this instead `'BUILD_INFO' or may try removing altogether as we are already passing in codeception.yml

Reply:
@pavanpatil1 Can you please try removing this as codeception.yml already has this

Reply:
@juhi123 means 'build' and 'project' ?..

Reply:
yes @pavanpatil1